src/openrct2/platform/Crash.cpp

```diff
@@ -48,12 +48,11 @@ const wchar_t* _wszArchitecture = WSZ(OPENRCT2_ARCHITECTURE);
 // so just hope the file name with '.gz' suffix is enough.
 // For docs on uplading to backtrace.io check
 // https://documentation.backtrace.io/product_integration_minidump_breakpad/
-static bool UploadMinidump(const wchar_t* dumpPath, int& error, std::wstring& response)
+static bool UploadMinidump(std::map<std::wstring, std::wstring> files, int& error, std::wstring& response)
```

Comment:
`const std::map<std::wstring, std::wstring> &files`

Reply:
I know, I think I left it this way here for a reason, but looking at the code now I can't seem to remember what it was exactly. I think in one of the revisions I had `UploadMinidump` add the dump path, instead of the caller.